packages/fiber/package.json

```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.17.8",
-    "@types/react-reconciler": "^0.28.9",
+    "@types/react-reconciler": "^0.32.0",
```

Comment:
@krispya with your latest commit, downgrading to ^0.31.0 => shouldn't types match too?

Reply:
I can try, the types previously were incomplete. And the current ones are incomplete too tbh.

Reply:
I checked and v0.31 has incorrect types. I am sticking with 0.32 as it has the most correct even if not perfectly aligned.